migrations/V3__replace_view.sql

```diff
@@ -0,0 +1,85 @@
+create or replace view
+  public.kpi_values_period_standardized as
+select distinct
+  on (
+    kd.kpi_id,
+    (
+      case
+        when kd.periodicity = 'yearly'::periodicity then date_trunc(
```

Comment:
Wow, that's a long one :D. Maybe we could get rid of the repetition by replacing the whole `case` statement by declaring it once and using `standardized_date` elsewhere?